cider/include/cider/batch/ScalarBatch.h

```diff
@@ -85,4 +85,59 @@ class ScalarBatch final : public CiderBatch {
   }
 };
 
+template <>
+class ScalarBatch<bool> final : public CiderBatch {
```

Comment:
will this class be conflict to this change? https://github.com/intel/BDTK/pull/104/files#diff-2e23a8d5af6d6d4a8668b9f372249cb3bfb8d450c124df95964ee50110d6c42aR39

Reply:
Oh, didn't notice the change in #104. Seems this one can be removed.